members/sophiePark.md

```diff
@@ -0,0 +1,15 @@
+# Sophie Se Hyun PARK
+
```

Comment:
We should add some description about us as well here

Reply:
Reflected in commit!